pytorch/buildspec.yml

```diff
@@ -68,44 +68,44 @@ images:
   #   device_type: &DEVICE_TYPE gpu
   #   python_version: &DOCKER_PYTHON_VERSION py3
   #   tag_python_version: &TAG_PYTHON_VERSION py38
-  #   cuda_version: &CUDA_VERSION cu113
+  #   cuda_version: &CUDA_VERSION cu116
   #   os_version: &OS_VERSION ubuntu20.04
   #   tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION, "-ec2" ]
   #   docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile.,
   #                        *DEVICE_TYPE ]
   #   target: ec2
   #   context:
   #     <<: *TRAINING_CONTEXT
-  BuildSageMakerCPUPTTrainPy3DockerImage:
-    <<: *TRAINING_REPOSITORY
-    build: &PYTORCH_CPU_TRAINING_PY3 false
-    image_size_baseline: 5000
-    # base_image_name: BuildEC2CPUPTTrainPy3DockerImage
-    device_type: &DEVICE_TYPE cpu
-    python_version: &DOCKER_PYTHON_VERSION py3
-    tag_python_version: &TAG_PYTHON_VERSION py38
-    os_version: &OS_VERSION ubuntu20.04
-    tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION, "-sagemaker" ]
-    docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
-    target: sagemaker
-    context:
-      <<: *TRAINING_CONTEXT
-  BuildSageMakerGPUPTTrainPy3DockerImage:
-    <<: *TRAINING_REPOSITORY
-    build: &PYTORCH_GPU_TRAINING_PY3 false
-    image_size_baseline: 14000
-    # base_image_name: BuildEC2GPUPTTrainPy3DockerImage
```

Comment:
should keep this `#`, or we can remove the `base_image_name` attribute since SM is no longer based on ec2